src/components/Firebase/firebase.js

```diff
@@ -17,13 +18,17 @@ class Firebase {
 
     /* Helper */
 
-    this.serverValue = app.database.ServerValue;
+    this.fieldValue = app.firestore.FieldValue;
     this.emailAuthProvider = app.auth.EmailAuthProvider;
 
     /* Firebase APIs */
 
     this.auth = app.auth();
-    this.db = app.database();
+
+    /* FireStore */
+    this.db = app.firestore();
+    const settings = { timestampsInSnapshots: true};
```

Comment:
Do we need this for the `serverTimestamp `?

Reply:
Yes, otherwise we have a warning in console from Firebase API

Reply:
```
index.js:1452 [2018-12-08T07:01:35.413Z]  @firebase/firestore: Firestore (5.5.9): 
The behavior for Date objects stored in Firestore is going to change
AND YOUR APP MAY BREAK.
To hide this warning and ensure your app does not break, you need to add the
following code to your app before calling any other Cloud Firestore methods```